frontend/src/Page.js

```diff
@@ -21,6 +22,7 @@ function Page({backend}) {
   const [scholiaMetadata, setScholiaMetadata] = useState([]);
   const [content, setContent] = useState([]);
   const [lastUpdate, setLastUpdate] = useState();
+  const [isCollection, setIsCollection] = useState(true);
```

Comment:
Do you really a need a state for this?
What does it mean exactly?

Reply:
We have tried the simple variable, but it not works so that we have to use useState.